coderd/agentapi/stats_test.go

```diff
@@ -39,7 +39,7 @@ type statsBatcher struct {
 	lastStats       *agentproto.Stats
 }
 
-var _ agentapi.StatsBatcher = &statsBatcher{}
+var _ workspacestats.StatsBatcher = &statsBatcher{}
```

Comment:
nit: should we rename the interface to get rid of the stuttering? I don't have any concrete suggestions though and don't want to block this just because of naming.